cloudsql/mysql/database-sql/cloudsql_test.go

```diff
@@ -0,0 +1,59 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"bytes"
+	"net/http/httptest"
+	"strings"
+	"testing"
+)
+
+func TestIndex(t *testing.T) {
+	app := newApp()
+	rr := httptest.NewRecorder()
+	request := httptest.NewRequest("GET", "/", nil)
+	app.indexHandler(rr, request)
+	resp := rr.Result()
+	body := rr.Body.String()
+
+	if resp.StatusCode != 200 {
+		t.Errorf("Expected StatusCode of 200. Got %d", resp.StatusCode)
```

Comment:
Please print got and want: https://github.com/golang/go/wiki/CodeReviewComments#useful-test-failures

Reply:
Are you looking to print "200" from a variable, or are you looking for a specific structure of the output, aka, exactly `"StatusCode == 500, want 200"'`?

Reply:
Yeah, got before want. Sorry that was unclear. Something like `indexHandler got status code %d, want 200`.